modules.json

```diff
@@ -7,82 +7,85 @@
                 "nf-core": {
                     "bcftools/annotate": {
                         "branch": "master",
-                        "git_sha": "911696ea0b62df80e900ef244d7867d177971f73",
-                        "installed_by": ["modules"]
+                        "git_sha": "44096c08ffdbc694f5f92ae174ea0f7ba0f37e09",
+                        "installed_by": ["modules"],
+                        "patch": "modules/nf-core/bcftools/annotate/bcftools-annotate.diff"
                     },
                     "bcftools/index": {
                         "branch": "master",
-                        "git_sha": "911696ea0b62df80e900ef244d7867d177971f73",
+                        "git_sha": "44096c08ffdbc694f5f92ae174ea0f7ba0f37e09",
                         "installed_by": [
-                            "multiple_impute_glimpse2",
                             "modules",
+                            "multiple_impute_glimpse2",
                             "vcf_impute_glimpse",
                             "vcf_phase_shapeit5"
                         ]
                     },
+                    "bcftools/mpileup": {
+                        "branch": "master",
+                        "git_sha": "44096c08ffdbc694f5f92ae174ea0f7ba0f37e09",
+                        "installed_by": ["modules"],
+                        "patch": "modules/nf-core/bcftools/mpileup/bcftools-mpileup.diff"
+                    },
                     "bcftools/norm": {
                         "branch": "master",
-                        "git_sha": "0435e4eebc94e53721c194b2d5d06f455a79e407",
+                        "git_sha": "44096c08ffdbc694f5f92ae174ea0f7ba0f37e09",
                         "installed_by": ["modules"]
                     },
                     "bcftools/query": {
                         "branch": "master",
-                        "git_sha": "b5cc8edd887e27c51a48387e34549d4405d9769f",
+                        "git_sha": "44096c08ffdbc694f5f92ae174ea0f7ba0f37e09",
                         "installed_by": ["modules"]
                     },
                     "bcftools/view": {
                         "branch": "master",
-                        "git_sha": "911696ea0b62df80e900ef244d7867d177971f73",
-                        "installed_by": ["modules"]
+                        "git_sha": "1013101da4252623fd7acf19cc581bae91d4f839",
+                        "installed_by": ["modules"],
+                        "patch": "modules/nf-core/bcftools/view/bcftools-view.diff"
                     },
                     "bedtools/makewindows": {
                         "branch": "master",
-                        "git_sha": "911696ea0b62df80e900ef244d7867d177971f73",
+                        "git_sha": "3b248b84694d1939ac4bb33df84bf6233a34d668",
                         "installed_by": ["vcf_phase_shapeit5"]
                     },
                     "custom/dumpsoftwareversions": {
```

Comment:
not deleted yet?

Reply:
@maxulysse what do you mean by not deleted yet ?